Add information about FeederCan (#58)
eclipse-velocitas/velocitas-docs: content/en/docs/tutorials/vehicle-app-runtime/run_runtime_services_locally.md

A CLI tool is provided for the interact with a running instance of the KUKSA Data Broker. It can be started by running the task `Local - VehicleDataBroker CLI`(by pressing _F1_, type _Run Task_ followed by `Local - VehicleDataBroker CLI`). The _KUKSA Data Broker_ needs to be running for you to be able to use the tool.

## Using KUKSA FeederCan

_FeederCan_ is a provider of a certain set of data points to the data broker.
To run _FeederCan_ as task please use `[F1 -> Tasks: Run Task -> Local - FeederCan]` and it will be run as a docker container.

> By default it will be use same file, that used for k3d environment: ```deploy/runtime/k3d/volume/dbcfileDefault.dbc```

Reviewer: ```suggestion
By default it will use the same file, that is used for the k3d environment: ```deploy/runtime/k3d/volume/dbcfileDefault.dbc```
```
